sklearn/model_selection/_validation.py

```diff
@@ -169,7 +169,7 @@ def cross_validate(
 
     scoring : str, callable, list, tuple, or dict, default=None
         Strategy to evaluate the performance of the cross-validated model on
-        the test set.
+        the test set. If None, the `estimator` object's `score` method is used.
```

Comment:
Good catch! I would word it something more inline with the definition of the [Estimator score method](https://scikit-learn.org/stable/modules/model_evaluation.html) and maybe even link to that page of the user guide. WDYT of something similar to:
> If None, the default evaluation criterion of the `estimator` is used.

Reply:
Yup, that's a good idea, it gives more info/context as the user does not necessarily know what 'score' method implies. Changed.

Reply:
Yes, I think @ArturoAmorQ's recommendation is better!